build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -57,7 +57,8 @@ jobs:
         $fullpackagePath = "${{ parameters.fullnupkgdir }}"
         Copy-Item -Path "$targetsFilePath\ProjectReunion-Nuget-Native.targets" -Destination "$fullpackagePath\build\native\Microsoft.ProjectReunion.Foundation.targets"
         Copy-Item -Path "$targetsFilePath\Microsoft.ProjectReunion.Foundation.props" -Destination "$fullpackagePath\build\native\Microsoft.ProjectReunion.Foundation.props"
-
+        Copy-Item -Path "$targetsFilePath\AppxManifest.xml" -Destination "$fullpackagePath\AppxManifest.xml"
```

Comment:
instead of piggybacking on this task that's specific to targets, either:
- update the comment and displayName
- create a separate task, similar to 'copy license to full nuget'